src/db.c

```diff
@@ -167,7 +86,7 @@ void updateLFU(robj *val) {
  * expired on replicas even if the master is lagging expiring our key via DELs
  * in the replication link. */
 robj *lookupKey(redisDb *db, robj *key, int flags) {
-    dictEntry *de = dbFind(db, key->ptr, DB_MAIN);
+    dictEntry *de = daFind(db->keys, key->ptr, getKeySlot(key->ptr));
```

Comment:
i think we should still have dbFind, that's a wrapper over daFind, and implicilty finds the slot (dict index).

Reply:
should i wrap all `da*` APIs that take a slot with a `db*` function that calls `getKeySlot`?

Reply:
not sure about all, but certainly the common ones. (find, and maybe add and remove)